lib/fsm/testhelpers.go

```diff
@@ -55,10 +56,11 @@ func (t *testEngine) GetExecutor(p ExecutorParams, r Remote) (PhaseExecutor, err
 // ChangePhaseState records the provided phase state change in the test engine.
 func (t *testEngine) ChangePhaseState(ctx context.Context, ch StateChange) error {
 	t.changelog = append(t.changelog, storage.PlanChange{
-		ID:       uuid.New(),
 		PhaseID:  ch.Phase,
 		NewState: ch.State,
-		Created:  time.Now().UTC(),
+		// Using real time on purpose, because the final state is determined
```

Comment:
You can use/set explicit timestamps though - it is still better than random data in a test.

Reply:
@a-palchikov Yeah, good point. I'll see if I can find an easy way to generate these fixed timestamps.